_includes/swc/setup.html

```diff
@@ -87,24 +87,32 @@ <h3>The Bash Shell</h3>
       </article>
       <article role="tabpanel" class="tab-pane active" id="shell-macos">
         <p>
-          The default shell prior to macOS 10.15 (Catalina) is Bash, and
+          The default shell in some versions of macOS is Bash, and
 	  Bash is available in all versions, so no need to install anything.
 	  You access Bash from the Terminal (found in
 	  <code>/Applications/Utilities</code>).
           See the Git installation <a href="https://www.youtube.com/watch?v=9LQhwETCdwY ">video tutorial</a>
           for an example on how to open the Terminal.
           You may want to keep Terminal in your dock for this workshop.
-	  If you are using macOS 10.15 (or later), you can run Bash by opening a
-          terminal and typing <code>bash</code>.
+	</p>
+	<p>
+          To see if your default shell is bash type <code>echo $SHELL</code> in
```

Comment:
"is Bash" (capitalized) instead of "is bash"? Would also apply to line 111 of this commit.

Reply:
Sorry about that, corrected - thanks for the review!